### PR TITLE
Feature: modal theme and id

### PR DESCRIPTION
**Pull request by ScottAntonacAP**
## Summary of Changes

This PR aims to allow the "more information" modal to be modified by either a theme option or an id. It also allows for CBT modal. Changes made
- Updated readme
- New class `MoreInfoOptions` that takes `modalTheme`, `modalId` and `isCbtEnabled`
- Updated example app demonstrating implementation

## Submission Checklist

- [x] Documentation is changed or added. (see [More Info Options](https://github.com/afterpay/sdk-android/blob/66713ee63916f240d67cf020af1511c7b0683eb6/README.md#more-info-options))

## Screenshots

All screenshots have been taken with merchant locale set to `en_AU`

<details>
<summary><code>moreInfoOptions</code> not set</summary>

</details>

<details>
<summary>Modal theme white</summary>

<pre><code>
afterpayBreakdown.moreInfoOptions = AfterpayMoreInfoOptions(
    modalTheme = AfterpayModalTheme.WHITE
)
</code></pre>

</details>

<details>
<summary>Cross border trade enabled</summary>

<pre><code>
afterpayBreakdown.moreInfoOptions = AfterpayMoreInfoOptions(
    isCBT = true
)
</code></pre>

</details>

<details>
<summary>Modal theme white and CBT enabled</summary>

<pre><code>
afterpayBreakdown.moreInfoOptions = AfterpayMoreInfoOptions(
    modalTheme = AfterpayModalTheme.WHITE,
    isCBT = true
)
</code></pre>

</details>

<details>
<summary>Modal ID set to 'en_US'</summary>

Setting a different locale with the modal ID should not be done in production, rather this is a demonstration that modal ID can be used.

<pre><code>
afterpayBreakdown.moreInfoOptions = AfterpayMoreInfoOptions(
    modalId = "en_US"
)
</code></pre>

</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -338,6 +338,25 @@ afterpayBreakdown.introText = AfterpayIntroText.MAKE_TITLE
 
 Given the above, the price breakdown text will be rendered `Make 4 interest-free payments of $##.##`
 
+##### More Info Options
+Setting `moreInfoOptions` is optional and of type `AfterpayMoreInfoOptions`. This class takes three parameters:
+- `modalId`: a `string` that is the filename of a modal hosted on Afterpay static.
+- `modalTheme`: an enum of type `AfterpayModalTheme` withe the following options: `DEFAULT`, `DEFAULT_CBT`, `WHITE` and `WHITE_CBT`.
```
Comment (luvgangwani92) [inline README.md:344]: Small typo.

```suggestion
- `modalTheme`: an enum of type `AfterpayModalTheme` with the following options: `DEFAULT`, `DEFAULT_CBT`, `WHITE` and `WHITE_CBT`.
```

**Comment by luvgangwani92** [comment]
@ScottAntonacAP - Is it possible to add some screenshots/screen recordings to the PR showing the working? I think it'll help visualize the changes better for someone revisiting.

File: README.md (hunk 1 above)
Comment (luvgangwani92) [reply README.md:344]: Also, is it possible to make this consistent with `ei-lib-js` where `data-modal-theme` supports `MINT` and `WHITE` and cbt modals are loaded based on the value (`true`/`false`) set for the `data-is-cbt` attribute? Ignore this comment if it's not possible technically.

Comment (ScottAntonacAP) [reply README.md]: Thanks for these two pickups. Committed the typo but the functionality for `isCBT` is how it works per the next line in the documentation. The above line was the way it was initially implemented and just wasn't removed. So fix will be to update the docs.

Comment (ScottAntonacAP) [reply README.md]: addressed in 8e71181

**Comment by ScottAntonacAP** [comment]
> @ScottAntonacAP - Is it possible to add some screenshots/screen recordings to the PR showing the working? I think it'll help visualize the changes better for someone revisiting.

Requested screenshots have been added along with example code.

**Review by luvgangwani92 — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -338,6 +338,28 @@ afterpayBreakdown.introText = AfterpayIntroText.MAKE_TITLE
...
+Setting `moreInfoOptions` is optional and of type `AfterpayMoreInfoOptions`. This class has two constructors.
+The first takes a single parameter:
+- `modalId`: a `string` that is the filename of a modal hosted on Afterpay static.
+
+The second takes two parameters:
+- `modalTheme`: an enum of type `AfterpayModalTheme` with the following options: `MINT` (default) and `WHITE`.
+- `isCbtEnabled`: a `boolean` to indicate if the modal should show the Cross Border Trade details in the modal
+
+**Notes**
+If both `modalId` is set `modalTheme` and `isCbtEnabled` are ignored.
```
Comment (benfterpay) [inline README.md:351]:
```suggestion
If `modalId` is set, both `modalTheme` and `isCbtEnabled` are ignored.
```

**Review by benfterpay — CHANGES_REQUESTED** [review]
Just one suggested tweak to the readme. Otherwise looks good to me.

File: README.md (hunk 2 above)
Comment (ScottAntonacAP) [reply README.md]: Committed this and realised that note is no longer applicable as due to constructor overloading, the class cannot be initialised with `modalId` and `modalTheme` / `isCbtEnabled`. Line removed as of f37fc22

**Review by benfterpay — APPROVED** [review]

**Pull request merged**